### PR TITLE
resolve #204, #206: support option keys with multi rows

### LINKED ISSUE #204: The keyboard rendering failed for Zinc keyboard
When rendering the Zinc keyboard, the following error occurred.

```
react-dom.production.min.js:216 TypeError: Cannot read property '0' of undefined
    at e.value (KeyboardModel.ts:238)
    at new e (KeyboardModel.ts:165)
    at new n (Keyboards.tsx:34)
    at yo (react-dom.production.min.js:135)
    at $a (react-dom.production.min.js:181)
    at Hu (react-dom.production.min.js:269)
    at Ts (react-dom.production.min.js:250)
    at _s (react-dom.production.min.js:250)
    at ks (react-dom.production.min.js:250)
    at ms (react-dom.production.min.js:243)
```

### PR DESCRIPTION
**Pull request by adamrocker**
Support multi rows optional keys and set max-width/min-width to keycodes' area.
スクリーンショット 2021-01-21 1 52 05
スクリーンショット 2021-01-21 1 52 18

### CODE REVIEW

File: src/actions/actions.ts [hunk 1]
```diff
@@ -265,10 +265,10 @@ export const AppActions = {
       },
     };
   },
-  updateKeyboardHeight: (height: number) => {
+  updateKeyboardSize: (width: number, height: number) => {
     return {
-      type: APP_UPDATE_KEYBOARD_HEIGHT,
-      value: height,
+      type: APP_UPDATE_KEYBOARD_SIZE,
+      value: { height, width },
```
Comment (yoichiro) [inline src/actions/actions.ts:271]:
```suggestion
      value: { width, height },
```

Of course, your code works, but I think that the pair "width" and "height" should be lined up as "width, height".

File: src/models/KeyboardModel.ts [hunk 2]
```diff
@@ -97,7 +98,11 @@ class KeymapItem {
   }
 
   get isDefault(): boolean {
-    return this.option == OPTION_DEFAULT;
+    return this.option == OPTION_DEFAULT || this.choice == '0';
```
Comment (yoichiro) [inline src/models/KeyboardModel.ts:101]:
```suggestion
    return this.option == OPTION_DEFAULT || this.isOrigin;
```

I think that the above code will become more understandable code.

**Review by yoichiro — CHANGES_REQUESTED** [review]
@adamrocker I posted a few comments. Could you confirm them? Thanks!

**Comment by monksoffunk** [comment]
Amazingly perfect for Zinc!
Thanks!!

**Pull request merged**